### PR TITLE
[#62] Use updated hsmodels

### PR DESCRIPTION
**Pull request by pkdash**

### CODE REVIEW

**Comment by pkdash** [comment]
Tests failed as hydroshare beta was down.

**Comment by sblack-usu** [comment]
retest this, please

**Comment by pkdash** [comment]
Tests are failing as beta is down.

**Comment by pkdash** [comment]
Tests ran successfully today!

File: docs/examples/Aggregation_Data_Object_Operations.ipynb [hunk 1]
```diff
@@ -858,47 +859,70 @@
...
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "# retrieve the updated georaster aggregation to verify the data was updated\n",
     "\n",
     "# need to first download this updated/new aggregation\n",
+    "download_to = r\"D:\\Temp\\GeoRaster_Testing\"\n",
```
Comment (sblack-usu) [inline docs/examples/Aggregation_Data_Object_Operations.ipynb:923]: Is this intentional? This looks like a Windows file path.

**Review by sblack-usu — APPROVED** [review]

File: docs/examples/Aggregation_Data_Object_Operations.ipynb (hunk 1 above)
Comment (pkdash) [reply docs/examples/Aggregation_Data_Object_Operations.ipynb]: Seems like left over from testing code. Fixed these file paths.

**Pull request merged**